views.py

```diff
@@ -450,6 +457,118 @@ def eventCheckInOut(request, event_id):
         return render(request, 'coderdojomobile/eventCheckInOut.html', context)
 
 
+def generate_uuid():
+    now = datetime.datetime.now()
+    rand = random.randint(1, 999)
+    micro = now.strftime("%f").zfill(6)[3:6]
+    uuid = now.strftime("%M%S") + str(micro) + str(rand).zfill(3)
+    return uuid
+
+
+def import_ticket(ticket, event):
+    '''
+    if badge id is not null, it is used to identify the participant
+    otherwise, name and surname are used
+    if the badge id does not match an existing badge id, a new participant
+    is created. This is done to allow Ninjas to use the same badge in more
+    than one Dojo.
+    No check is performed on name - surname w.r.t badge id.
+    '''
+    name = ticket['Name']
+    surname = ticket['Surname']
+    ticket_id = ticket['Ticket_id']
+    badge_id = ticket['Badge_id']
+    participant = None
+    if badge_id is not None and len(badge_id) > 0:
+        # Look for the Participant
+        try:
+            participant = Participant.objects.get(uuid=badge_id)
+        except Participant.DoesNotExist:
+            None
```

Comment:
Please use "pass" and not None if nothing has to be done. I would prefer an error message.